compiler/ilgen/OMRIlBuilder.cpp

```diff
@@ -2581,8 +2663,7 @@ OMR::IlBuilder::Switch(const char *selectionVar,
       cfg()->addEdge(switchBlock, caseBlock);
       AppendBuilder(handler);
 
-      TR::Node *caseNode = TR::Node::createCase(0, caseBlock->getEntry(), value);
-      lookupNode->setAndIncChild(c+2, caseNode);
+      switchNode->setAndIncChild(c+2, TR::Node::createCase(0, caseBlock->getEntry(), value));
```

Comment:
I actually prefer the way it was before (for debugging purposes), but I'll call it a nit.

Reply:
Yeah I agree.